src/handler/src/commands/github/issue-pr.ts

```diff
@@ -0,0 +1,453 @@
+
+// #region typings // TODO: remove section (indev)
+
+const BASE_URL = 'https://api.github.com/graphql';
+
+interface StringStringMapping {
+	[index: string]: string | undefined;
+}
+
+interface StringReviewMapping {
+	[index: string]: GitHubReview | undefined;
+}
+
+const RepositoryAliases: StringStringMapping = {
+	g: 'discord.js',
+	c: 'collection',
+	dapi: 'discord-api-docs',
+	next: 'discord.js-next',
+} as const;
+
+const LabelColors: StringStringMapping = {
+	'0075ca': '<:0075ca:751210299394359316>',
+	'027b69': '<:027b69:751210290846367825>',
+	'1d637f': '<:1d637f:751210249721217155>',
+	'276bd1': '<:276bd1:751210308500062310>',
+	'4b1f8e': '<:4b1f8e:751210257811767297>',
+	'7057ff': '<:7057ff:751210318385905724>',
+	'7ef7ef': '<:7ef7ef:751210266217414768>',
+	aed5fc: '<:aed5fc:751210330020905003>',
+	b6b1f9: '<:b6b1f9:751210340577968199>',
+	c10f47: '<:c10f47:751210349918683249>',
+	c66037: '<:c66037:751210359477764126>',
+	cfd3d7: '<:cfd3d7:751210367430033489>',
+	d73a4a: '<:d73a4a:751210378981015565>',
+	d876e3: '<:d876e3:751210389143814274>',
+	default: '<:default:751211609430425611>',
+	e4e669: '<:e4e669:751210418999001128>',
+	e4f486: '<:e4f486:751210427777679362>',
+	e8be8b: '<:e8be8b:751210441707094036>',
+	ea8785: '<:ea8785:751210466033795274>',
+	f06dff: '<:f06dff:751210476523749487>',
+	fbca04: '<:fbca04:751210487508762675>',
+	fc1423: '<:fc1423:751210498950955048>',
+	fcf95a: '<:fcf95a:751210515203620928>',
+	ffccd7: '<:ffccd7:751210528021544991>',
+	ffffff: '<:ffffff:751210537597272076>',
+} as const;
+
+enum ResultStatePR {
+	OPEN = 'OPEN',
+	CLOSED = 'CLOSED',
+	MERGED = 'MERGED',
+	DRAFT = 'DRAFT',
+}
+
+enum ResultStateIssue {
+	OPEN = 'OPEN',
+	CLOSED = 'CLOSED',
+}
+
+enum InstallableState {
+	OPEN = 'OPEN',
+	DRAFT = 'Draft',
+}
+
+enum StateColors {
+	OPEN = 4827469,
+	CLOSED = 12267569,
+	MERGED = 6441376,
+	DRAFT = 12961221,
+}
+
+const Timestamps = {
+	OPEN: 'publishedAt',
+	CLOSED: 'closedAt',
+	MERGED: 'mergedAt',
+	DRAFT: 'publishedAt',
+} as const;
+
+type TimestampsWithoutMerged = Omit<typeof Timestamps, 'MERGED'>;
+
+type TimestampsWithoutMergedKey = TimestampsWithoutMerged[keyof TimestampsWithoutMerged];
+
+enum TickStates {
+	TICK = '<:tick:747502128003809332>',
+	NO_TICK = '<:notick:751200038323093521>',
+}
+
+enum AllowedRepositories {
+	'action-docs',
+	'action-eslint',
+	'action-webpack',
+	'collection',
+	'commando',
+	'discord-api-docs',
+	'discord-api-types',
+	'discord.js-next',
+	'discord.js',
+	'erlpack',
+	'form-data',
+	'guide',
+	'node-pre-gyp',
+	'opus',
+	'rpc',
+	'webhook-filter',
+	'website',
+}
+
+enum PRIcons {
+	OPEN = 'https://cdn.discordapp.com/emojis/751210109333405727.png',
+	CLOSED = 'https://cdn.discordapp.com/emojis/751210080459817092.png',
+	MERGED = 'https://cdn.discordapp.com/emojis/751210169609748481.png',
+	DRAFT = 'https://cdn.discordapp.com/emojis/751210097463525377.png',
+}
+
+enum IssueIcons {
+	OPEN = 'https://cdn.discordapp.com/emojis/751210140086042686.png?v=1',
```

Comment:
```suggestion
	OPEN = 'https://cdn.discordapp.com/emojis/751210140086042686.png',
```
consistency with other urls

Reply:
urls and emojis will change before merge anyways